src/displayapp/screens/WatchFaceDigital.h

```diff
@@ -7,6 +7,7 @@
 #include "displayapp/screens/Screen.h"
 #include "components/datetime/DateTimeController.h"
 #include "components/ble/BleController.h"
+#include "components/notifications/NotificationManager.h"
```

Comment:
the `NotificationManager` class is forward declared below. So we need either the include or the forward declaration below, but not both